platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/schedulers/internal/SequentialTaskScheduler.java

```diff
@@ -83,16 +82,16 @@ public SequentialTaskScheduler(
         this.offRamp = Objects.requireNonNull(offRamp);
         this.busyTimer = Objects.requireNonNull(busyTimer);
 
-        this.nextTaskPlaceholder = new AtomicReference<>(
-                new SequentialTask(pool, offRamp, busyTimer, uncaughtExceptionHandler, getSquelcher(), true));
+        this.nextTaskPlaceholder =
+                new AtomicReference<>(new SequentialTask(pool, offRamp, busyTimer, uncaughtExceptionHandler, true));
     }
 
     /**
      * {@inheritDoc}
      */
     @Override
     protected void put(@NonNull final Consumer<Object> handler, @NonNull final Object data) {
-        if (getSquelcher().shouldSquelch()) {
+        if (currentlySquelching()) {
             return;
         }
```

Comment:
For this scheduler (and for the other types as well), I'd suggest we drop the squelch checks for put/offer/inject, and insert into the queue regardless of squelching status. It causes us to do extra work while in squelching mode, but it saves us from having to check the atomic boolean once per object during normal operation.

Reply:
Done